src/envoy/http/service_control/config_parser.h

```diff
@@ -36,6 +38,10 @@ constexpr int64_t kDefaultMinStreamReportIntervalMs = 10000;
 constexpr int64_t kLowerBoundMinStreamReportIntervalMs = 100;
 }  // namespace
 
+// The filter name.
+const std::string ServiceControlFilterName =
```

Comment:
go/totw/140

I think we build with c++ 17 now, right? Then `inline` should work.

```
inline constexpr absl::string_view kFilterName
```

Reply:
No, in my local, we are not using c++17